### PR TITLE
Fix spelling and change tests

### PR DESCRIPTION
**Pull request by mbounabi**
change "ASCENCEUR" to "ASCENSEUR"

### CODE REVIEW

File: source/jormungandr/jormungandr/equipments/tests/sytral_equipment_test.py [hunk 1]
```diff
@@ -64,23 +64,23 @@ def equipments_get_information_test():
...
     # equipment_details is not added
-    st = create_stop_point("TCL_ASCENCEUR", "262")
+    st = create_stop_point("TCL_ASCENSEUR", "262")
     provider.get_informations_for_journeys([st])
     assert not st.equipment_details
 
     # stop point has code with incorrect type but value present in webservice response
     # equipment_details is not added
-    st = create_stop_point("ASCENCEUR", "261")
+    st = create_stop_point("TCL_ASCENCEUR", "261")
```
Comment (pbougue) [inline source/jormungandr/jormungandr/equipments/tests/sytral_equipment_test.py:84]: Wasn't `TCL_` removed on purpose, before?
Looks like it from the comment.

**Review by pbougue — COMMENTED** [review]
The rest is OK.

**Review by kinnou02 — APPROVED** [review]

File: source/jormungandr/jormungandr/equipments/tests/sytral_equipment_test.py (hunk 1 above)
Comment (mbounabi) [reply source/jormungandr/jormungandr/equipments/tests/sytral_equipment_test.py:84]: The test case is to have an incorrect code, so now "TCL_ASCENCEUR" is incorrect ;)

Comment (pbougue) [reply source/jormungandr/jormungandr/equipments/tests/sytral_equipment_test.py:84]: Aaaaaah :stuck_out_tongue_closed_eyes:
OK, nice!

**Review by pbougue — APPROVED** [review]

**Pull request merged**